jin/7주차 과제/생일.py

```diff
@@ -0,0 +1,11 @@
+n = int(input())
+students = []
+for i in range(n):
+    temp_students = input().split(' ') 
```

Comment:
```suggestion
    name, datem month, year= input().split() 
```
이 형식으로 받아서 배열에 (year,month, date, name)를 삽입한 후에 정렬하는 방법도 있을 것 같아요, 람다식을 활용하는 풀이도 좋네요. 수고하셨습니다~